crypto/crypto/src/hash.rs

```diff
@@ -62,16 +62,18 @@
 //! For any new structure `MyNewStruct` that needs to be hashed, it is recommended to simply
 //! use the derive macro [`CryptoHasher`](https://doc.rust-lang.org/reference/procedural-macros.html).
 //!
-//! ```ignore
-//! #[derive(CryptoHasher)]
-//! struct MyNewStruct {
-//!   ...
-//! }
+//! ```
+//! # use libra_crypto_derive::CryptoHasher;
+//! # use serde::Deserialize;
+//! #[derive(Deserialize, CryptoHasher)]
```

Comment:
same comment on ordering

Reply:
That one, I'm going to pass because afaik rustfmt does not care.